.eslintrc.json

```diff
@@ -10,6 +10,8 @@
   "rules": {
     "prettier/prettier": [
       "error"
-    ]
+    ],
```

Comment:
Not sure, if this PR should change the eslint config

Reply:
I think it's not required, I'd to do that to make es lint test pass from my end. 

https://github.com/adonisjs/core/discussions/2242

Reply:
Instead of removing the rule. I recommend fix the error.

Reply:
Please have a look at the error, I know a very little about it

```

/usr/src/app/example/index.ts
  5:6  error  'ProfileTypes' is already declared in the upper scope on line 5 column 6  no-shadow

/usr/src/app/src/Factory/FactoryModel.ts
  45:9  error  'Model' is already declared in the upper scope on line 37 column 27  no-shadow
```


I think the error is not because of my update, as in pipeline I see others getting same error. 

If I revert back this lines, my build will fail on circle. Please confirm

Reply:
@thetutlage 

Fixed: I've modified example/index.ts and src/Factory/FactoryModel.ts to pass on eslint test. So now build passes 👍